test_data/series7_device_config.yaml

```diff
@@ -1,9 +1,28 @@
 # Which BEL names are global buffers for nextpnr?
-global_buffers:
+global_buffer_bels:
 - BUFG
 - BUFGCTRL
 - VCC
 - GND
+# Which cell names are global buffers, and which pins should use dedicated routing resources
+global_buffer_cells:
+  - cell: BUFG
+    pins: # list of pins that use global resources
+     - name: I # pin name
+       guide_placement: true # attempt to place so that this pin can use dedicated resources
+       max_hops: 10 # max hops of interconnect to search
```

Comment:
I suppose this number comes from some experiments, might be worth adding some comments around where this comes from.

I am not sure I understand why only the input pins have the max_hops defined.

Reply:
It was just a test of the description, and its for inputs only as we only want to put so much effort into finding locations for those with dedicated routing resources.